### DISCUSSION TITLE
CYGWIN

### DISCUSSION THREAD

**Discussion by alycest** [HathewayWill/WRF-MOSIT#20] [General]
Hey, will this work on CYGWIN_NT-10.0-WOW operating system? 
We have a remote modelling computer, which may be able to handle WRF.
I just can't get the script running on the modelling PC.
Thanks, A

**Comment by HathewayWill** [comment]
Good morning,

So the script is Debian based OS only at this time and since cygwin is red
hat Linux based it won't work on it.

I'm currently developing at red hat based install but it's on temporary
hold as I recover from surgery.

**Reply by alycest** [reply]
No worries. Don't sit to much at your computer then!

**Comment by HathewayWill** [comment]
Direct email me I'll send you something that cacn help with that though.
